content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -185,7 +185,7 @@ To configure an validating admission policy for use in a cluster, a binding and
 are created. The following is an example of a ValidatingAdmissionPolicyBinding.
 
 ```yaml
```

Comment:
Could this use a [`codenew`](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files) shortcode?